pallets/automation-time/src/tests.rs

```diff
@@ -1,4 +1,4 @@
-use crate::{mock::*, Error, OverlflowTasks, Task, Tasks};
+use crate::{mock::*, Error, OverlflowTasks, Task, TaskHashInput, Tasks};
```

Comment:
what about a test for colliding `provided_id`'s

Reply:
that is in there: https://github.com/OAK-Foundation/OAK-blockchain/pull/58/files#diff-8aa2921b8ffd2c8a5901003639e03bb443a2032d62c687606ea6a9f7bbcaf99cR114